static/src/js/components/PublishPreview/__tests__/__mocks__/publishPreview.js

```diff
@@ -0,0 +1,497 @@
+export const publishPreview = {
+  relatedCollections: null,
+  abstract: 'Mock Testing Collections',
+  accessConstraints: null,
+  additionalAttributes: [
+    {
+      parameterUnitsOfMeasure: 'meters',
+      dataType: 'STRING',
+      parameterValueAccuracy: 'centimeter',
+      valueAccuracyExplanation: 'Mock Text ',
+      description: 'Mock Very Long Text Mock Very Text Mock Very Long Text Mock Very Long Text Mock Very LongText Mock Very Long Text Mock Very Long Text Mock Very Long Text  Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text  Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text  Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text  Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Text Mock Very Long Text Mock Very Long Text Mock Very LongText Mock Very Long Text Mock Very Long Text Mock Very Long Text  Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text  Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text  Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text  Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Long Text Mock Very Text Mock Very Long Text Mock Very Long Text Mock Very LongText Mock Very Long',
```

Comment:
Maybe use some Lorem Ipsum or a real description here 😆

Reply:
Ooop... This is what I get get for being lazy and getting a test record from SIT, SMH. Fixed.